runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -940,7 +939,12 @@ void chpl_comm_init(int *argc_p, char ***argv_p) {
   time_init();
   chpl_comm_ofi_oob_init();
   DBG_INIT();
+  chpl_oversubscribed = chpl_env_rt_get_bool("OVERSUBSCRIBED", false);
 
+  chpl_numLocalesOnNode = chpl_comm_ofi_oob_locales_on_node();
+  if (chpl_numLocalesOnNode > 1) {
+    chpl_oversubscribed = true;
+  }
```

Comment:
I think we want the env var to take precedence, and I think that isn't the case here if a user sets `CHPL_RT_ OVERSUBSCRIBED` but we detected oversubscription. Maybe something like:

```c
bool def_oversubscribed = chpl_comm_ofi_oob_locales_on_node() > 1;
chpl_oversubscribed = chpl_env_rt_get_bool("OVERSUBSCRIBED", def_oversubscribed);
```

Reply:
I thought setting `CHPL_RT_OVERSUBSCRIBED` indicated oversubscription. Are you saying the user can set `CHPL_RT_OVERSUBSCRIBED=false` to indicate the node is not oversubscribed?

Reply:
Correct, previously it didn't make sense to set `CHPL_RT_OVERSUBSCRIBED=false` since that was the default value, but now that it can be set automatically by the runtime I think we need a way to override.

Reply:
Ok, that wasn't clear to me. I can make that change.